algo-exercise/getting-started/src/test/java/codingblackfemales/gettingstarted/MyAlgo2Test.java

```diff
@@ -1,64 +1,49 @@
 package codingblackfemales.gettingstarted;
 
 import codingblackfemales.algo.AlgoLogic;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Moved everything to Junit5 for consistency. Also you can now run everything in Intellij as they are all teh same type.